src/ray/object_manager/plasma/store.cc

```diff
@@ -313,7 +313,8 @@ void PlasmaStore::ConnectClient(const boost::system::error_code &error) {
     // Accept a new local client and dispatch it to the node manager.
     auto new_connection = Client::Create(
         // NOLINTNEXTLINE : handler must be of boost::AcceptHandler type.
-        boost::bind(&PlasmaStore::ProcessMessage, this, ph::_1, ph::_2, ph::_3),
+        boost::bind(&PlasmaStore::ProcessClientMessage, this, ph::_1, ph::_2, ph::_3),
```

Comment:
I don't understand why `std::bind` is needed here, try lambda.
Reference: effective modern C++ item-34